lib/contributor.rb

```diff
@@ -0,0 +1,11 @@
+class Contributor
+  def initialize(file_path = 'CONTRIBUTORS.md')
+    @file_path = file_path
+  end
+
+  def add(contributor)
+    File.open(@file_path, 'a+') do |f|
+      f.puts "- [@#{contributor}] (https://github.com/#{contributor})"
```

Comment:
you're adding a space here, between `]` and `(`, this will make that not be a proper link :D